deny.toml

```diff
@@ -8,6 +8,7 @@ allow = [
   "Apache-2.0",
   "BSD-3-Clause",
   "Unicode-DFS-2016",
+  "bzip2-1.0.6",
```

Comment:
Link to the license

https://github.com/enthought/bzip2-1.0.6/blob/master/LICENSE